src/env/dev/terraform.tfvars

```diff
@@ -232,3 +233,11 @@ buyerbanks_function_autoscale_minimum = 1
 buyerbanks_function_autoscale_maximum = 3
 buyerbanks_function_autoscale_default = 1
 buyerbanks_delete_retention_days      = 30
+
+# proxy app service
+proxy_enabled        = true
+proxy_redis_capacity = 1
+proxy_redis_sku_name = "Standard"
```

Comment:
For DEV env, to change sku name in "Basic" --> https://azure.microsoft.com/en-au/pricing/details/cache/

p.s for example --> https://github.com/pagopa/selfcare-infra/blob/439a5a34bca237095eb7a58cb670eff03a41c5ab/src/core/env/dev/terraform.tfvars#L44